jwst/pipeline/calwebb_spec3.py

```diff
@@ -331,6 +335,18 @@ def process(self, input_data):
         if exptype in WFSS_TYPES:
             if self.save_results:
                 x1d_output = make_wfss_multiexposure(wfss_x1d)
+                # Populate S_REGION in first entry of output model spec list.
```

Comment:
It would be cleaner to either split this into a helper function, or else consume it into `make_wfss_multiexposure`.  I kind of prefer the second one but maybe that's not possible anymore after the list of x1d products has been generated (do they carry this S_REGION information?)